server/pom.xml

```diff
@@ -48,6 +48,12 @@
             <version>${project.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
![image](https://user-images.githubusercontent.com/8758457/57745085-5157f380-76fe-11e9-9c6b-a733f64215f9.png)

1. why i need seata-all exclusions releated seata, for example: seata-server not need seata-spring.
2. how to support registry? registration center like nacos can't run.
3. Why define the scope of indirect dependencies as compile time? Does httpcore work? Isn't it time to sort out the indirect dependency scope set to compile here?

Reply:
You can take a look at the content of seata-all jar, which contains all the Seata code, including seata-core, seata-spring and so on.

The 3rd jar's scope is 'provided' default which defined in bom.xml, change it to 'compile' so that they can be packed into the 'distribution/lib'.